extensions/wikia/PhalanxII/services/PhalanxService.class.php

```diff
@@ -233,22 +206,14 @@ private function sendToPhalanxDaemon( $action, $parameters ) {
 				}
 			}
 
-			// SUS-964: Give reload requests more time to succeed
-			// Reload requests are only sent upon saving/modifying a block,
-			// so using a higher value here won't affect site performance
-			if ( $action === 'reload' ) {
-				$options['timeout'] = static::PHALANX_SERVICE_RELOAD_TIMEOUT;
-			}
```

Comment:
Are we sure that the `reload` requests are now faster then they used to be?

Reply:
We now use RabbitMQ instead, so no need to set special timeout 😉

Reply:
Ah, true :)